packages/core/src/sync/fragments.ts

```diff
@@ -204,76 +126,149 @@ export const getTransactionFilterFragmentIds = ({
   return fragments;
 };
 
-export const getTransferFilterFragmentIds = ({
+export const getTraceFilterFragmentIds = ({
   chainId,
   fromAddress,
   toAddress,
-}: Omit<TransferFilter, "fromBlock" | "toBlock"> & {
+  callType,
+  functionSelector,
+  ...filter
+}: Omit<TraceFilter, "fromBlock" | "toBlock"> & {
   chainId: number;
 }): FragmentReturnType => {
   const fragments: FragmentReturnType = [];
   const fromAddressFragmentIds = getAddressFragmentIds(fromAddress);
   const toAddressFragmentIds = getAddressFragmentIds(toAddress);
+  const includeTransactionReceipts = shouldGetTransactionReceipt(filter);
 
   for (const fragmentFromAddress of fromAddressFragmentIds) {
     for (const fragmentToAddress of toAddressFragmentIds) {
-      const id =
-        `transfer_${chainId}_${fragmentFromAddress.id}_${fragmentToAddress.id}` as const;
+      for (const fragmentFunctionSelector of Array.isArray(functionSelector)
+        ? functionSelector
+        : [functionSelector]) {
+        const id =
+          `trace_${chainId}_${fragmentFromAddress.id}_${fragmentToAddress.id}_${callType ?? null}_${fragmentFunctionSelector ?? null}_${includeTransactionReceipts ? 1 : 0}` as const;
```

Comment:
Could you explain why `callType` is included in the fragment key and when it might come into play?

Reply:
Gonna remove